packages/dashboard/src/components/schedule-visualizer/index.tsx

```diff
@@ -195,17 +196,22 @@ export default function ScheduleVisualizer(props: ScheduleVisualizerProps): Reac
           },
         });
         debug('set trajectories');
-        setTrajectories((prev) => ({
-          ...prev,
-          [curMapFloorLayer.level.name]: resp,
-        }));
+        // negotiations panel index is 6 on the omni panel
+        if (omniPanelView === 6) {
```

Comment:
Can we use some sort of enum here to avoid hardcoding the index position?

Reply:
Re-factored out the enum